components/dialog/metadata/dialog.yml

```diff
@@ -262,9 +289,9 @@ examples:
     demoClassName: spectrum-CSSExample-example--overlay
     markup: |
       <button class="spectrum-Button spectrum-Button--sizeM spectrum-Button--outline spectrum-Button--staticWhite spectrum-CSSExample-overlayShowButton" onclick="openDialog(this.nextElementSibling)"><span class="spectrum-Button-label">Open Fullscreen Takeover Dialog</span></button>
-      <div class="spectrum-Modal-wrapper">
+      <div class="spectrum-Modal-wrapper spectrum-CSSExample-modal">
         <div class="spectrum-Modal spectrum-Modal--fullscreenTakeover" data-testid="modal">
-          <section class="spectrum-Dialog spectrum-Dialog--fullscreenTakeover" role="alertdialog" tabindex="-1" aria-modal="true">
+          <section class="spectrum-CSSExample-dialog spectrum-Dialog spectrum-Dialog--fullscreenTakeover" role="alertdialog" tabindex="-1" aria-modal="true">
```

Comment:
Do these two examples need the class `spectrum-CSSExample-dialog` added? I had seen that this example class adds a bit of CSS. I disabled those CSS declarations and it still appears the same.

Reply:
You might be right! I was just trying to use the class consistently, but it doesn't need it with the z-index fixed.

Reply:
It does seem a little funny not to use `.spectrum-CSSExample-dialog` in the Dialog component, but keep it around for styling wrappers in other places such as [alert dialog](https://github.com/adobe/spectrum-css/blob/main/components/alertdialog/metadata/alertdialog.yml#L10) and [menu](https://github.com/adobe/spectrum-css/blob/main/components/menu/metadata/menu.yml#L1351) as well as others.

But, being that we're planning to leave the docs site behind, it may not be worthwhile to look into it.